3_visualize/src/diagnostics.R

```diff
@@ -0,0 +1,34 @@
+#' Export model diagnostics
+#' @export model_diagnostics_text txt, file contains mean RSME for three model types as well as a breakdown of training temperature profiles for the different models 
+
+diagnostics <- function(){
+  
+  # Check for existence of out subfolder
+  dir.exists("3_visualize/out")
+  
+  # Read in eval data 
+  eval_data <- readRDS("2_process/out/eval_data")
+  
+  # Save the model diagnostics
+  render_data <- list(pgdl_980mean = filter(eval_data, model_type == 'pgdl', exper_id == "similar_980") %>% pull(rmse) %>% mean %>% round(2),
+                      dl_980mean = filter(eval_data, model_type == 'dl', exper_id == "similar_980") %>% pull(rmse) %>% mean %>% round(2),
+                      pb_980mean = filter(eval_data, model_type == 'pb', exper_id == "similar_980") %>% pull(rmse) %>% mean %>% round(2),
+                      dl_500mean = filter(eval_data, model_type == 'dl', exper_id == "similar_500") %>% pull(rmse) %>% mean %>% round(2),
+                      pb_500mean = filter(eval_data, model_type == 'pb', exper_id == "similar_500") %>% pull(rmse) %>% mean %>% round(2),
+                      dl_100mean = filter(eval_data, model_type == 'dl', exper_id == "similar_100") %>% pull(rmse) %>% mean %>% round(2),
+                      pb_100mean = filter(eval_data, model_type == 'pb', exper_id == "similar_100") %>% pull(rmse) %>% mean %>% round(2),
+                      pgdl_2mean = filter(eval_data, model_type == 'pgdl', exper_id == "similar_2") %>% pull(rmse) %>% mean %>% round(2),
+                      pb_2mean = filter(eval_data, model_type == 'pb', exper_id == "similar_2") %>% pull(rmse) %>% mean %>% round(2))
```

Comment:
This is a good chunk of repetitious code. I would like you to write a function to cut down on the copy-paste